scripts/check-dependencies.js

```diff
@@ -0,0 +1,21 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Red Hat, Inc. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+//check package.json do not have dependency with 'next' version
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const exit = require('process').exit;
+const dependencies = require('../package.json').dependencies;
+
+for (const dep in dependencies) {
+  if (Object.prototype.hasOwnProperty.call(dependencies, dep)) {
+    const version = dependencies[dep];
+    if (version === 'next') {
```

Comment:
This appears to be a general filter across all dependencies. I do not think we would want `next` as a version on anything but if we do, then this would prevent the release. 
Do we want it broad or specific?

Reply:
I do not think that release with `next` version is good idea, for any dependency. Because it is a tag, not a some specific version, and real version behind that tag may change, and you may ged unpredictable behaviour in future.

Reply:
I agree. I just want us to be intentional in what we do.